src/main/java/bowling/model/Pins.java

```diff
@@ -0,0 +1,65 @@
+package bowling.model;
+
+import bowling.ExceptionMessages;
+
+import java.util.Objects;
+
+public class Pins {
+    public static final int MIN_PINS = 0;
+    public static final int MAX_PINS = 10;
+
+    private final int countOfPins;
+
+    public Pins() {
+        countOfPins = MAX_PINS;
+    }
```

Comment:
```suggestion
    public Pins() {
        this(MAX_PINS);
    }
```
이렇게 표현할 수 있을 것 같아요!
혹은 아예 of에게 위임할 수도 있을 것 같아요 ㅎㅎ